docker/test/integration/MiNiFi_integration_test_driver.py

```diff
@@ -248,7 +248,7 @@ def check_query_results(self, postgresql_container_name, query, number_of_rows,
         assert self.cluster.check_query_results(postgresql_container_name, query, number_of_rows, timeout_seconds)
 
     def wait_for_container_logs(self, container_name, log_pattern, timeout_seconds, count=1):
-        self.cluster.wait_for_app_logs_regex(container_name, log_pattern, timeout_seconds, count)
+        assert self.cluster.wait_for_app_logs_regex(container_name, log_pattern, timeout_seconds, count)
```

Comment:
Good catch!

I would rename this function to `check_container_log_matches_regex`, as it does more than just wait for the logs.  Also, it is the more general pair of `check_minifi_log_matches_regex`, similarly to the pair of functions `check_minifi_log_contents` and `check_container_log_contents`.

Reply:
Done.